android/app/src/main/java/org/fossasia/openevent/activities/SearchActivity.java

```diff
@@ -190,7 +192,7 @@ public void addResultsFromLocations(String queryString) {
 
         filteredMicrolocations.addChangeListener(microlocations -> {
             if(microlocations.size()>0) {
```

Comment:
The formatting is not followed. There needs to be space after `if`. Its best to press `Ctrl + Alt + LShift + L` to format the entire file on the go.